### PR TITLE
Don't enable Apache site if native nginx support exists 

### PR DESCRIPTION
**Pull request by jvonau**
without proxy_pass to Apache

### CODE REVIEW

File: roles/rachel/tasks/rachel_enabled.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# FIXME
```
Comment (holta) [inline roles/rachel/tasks/rachel_enabled.yml:1]:
```suggestion
```
roles/rachel has been deprecated for years.

As far as I know, we don't want to fix it, as /library/www/html/modules works as is (enabled in other ways).

**Comment by jvonau** [comment]
supersedes #2102 with #2134 #2143

**Comment by holta** [comment]
@jvonau's listing @ https://github.com/iiab/iiab/issues/2078#issuecomment-568225872 also very helpful to understanding & evolving https://github.com/iiab/iiab/blob/master/roles/nginx/README.md over the coming days/weeks!

**Comment by jvonau** [comment]
So why can't I use the web interface to resolve conflicts like I was able to do in other PRs?

**Comment by holta** [comment]
> So why can't I use the web interface to resolve conflicts like I was able to do in other PRs?

It's happened to me too, for certain very simple merge conflicts.  Very annoying.

**Comment by jvonau** [comment]
I see why, files were renamed in 2166.

**Comment by jvonau** [comment]
superseded by #2189

**Pull request closed without merging**